### PR TITLE
exiv2 0.26

### PR DESCRIPTION
**Pull request by graphaelli**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by graphaelli** [comment]
pyexiv2 is deprecated per http://tilloy.net/dev/pyexiv2/

**Comment by fxcoudert** [comment]
`geeqie` and `gexiv2` both fail with: `dyld: Library not loaded: /usr/local/opt/exiv2/lib/libexiv2.14.dylib`

If the library name has changed, a new revision to these formulas are needed.

File: Formula/pyexiv2.rb [hunk 1]
```diff
@@ -1,43 +0,0 @@
-class Pyexiv2 < Formula
```
Comment (fxcoudert) [inline Formula/pyexiv2.rb]: If `pyexiv2` still builds, please keep it (even though it is deprecated).

**Review by fxcoudert — DISMISSED** [review]

**Comment by graphaelli** [comment]
The library name is the same, I wouldn't expect changes needed for those formula: 
```
/usr/local/opt/exiv2/lib/libexiv2.26.dylib
/usr/local/opt/exiv2/lib/libexiv2.a
/usr/local/opt/exiv2/lib/libexiv2.dylib -> libexiv2.26.dylib
```

thoughts?

**Comment by ilovezfs** [comment]
The library number changed from 2.14 to 2.26 so geeqie and gexiv2 need to be revision bumped.

**Comment by graphaelli** [comment]
Is there a way to trigger bottle rebuilds?  Both packages install from source without issue (both do `-lexiv2`) and pass `brew test` afterwards (and both failed with bottle installs).

**Comment by ilovezfs** [comment]
Yes, you add revisions like here: https://github.com/Homebrew/homebrew-core/pull/13021/files

**Comment by graphaelli** [comment]
@ilovezfs thanks for the pointer, revisions bumped and tests are passing
@fxcoudert pyexiv2 added back after bumping revision

File: Formula/pyexiv2.rb (hunk 1 above)
Comment (graphaelli) [inline Formula/pyexiv2.rb]: pyexiv2 added back after bumping revision

**Comment by ilovezfs** [comment]
@graphaelli Great work here! Thank you.

**Pull request closed without merging**